hs_tracking/models.py

```diff
@@ -0,0 +1,147 @@
+from json import loads, dumps
+from datetime import datetime, timedelta
+
+from django.db import models
+from django.core import signing
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from theme.models import UserProfile
+
+
+SESSION_TIMEOUT = 60 * 15
```

Comment:
@ironfroggy I am wondering whether it is better to put this session_timeout into settings.py, so it might be a bit easier to change when needed.